modules/govuk/manifests/apps/email_alert_service.pp

```diff
@@ -51,6 +52,10 @@
     password => $rabbitmq_password,
   }
 
+  govuk::app::envvar::redis { 'email-alert-service':
+    host => $redis_host,
```

Comment:
Do we need the port too?

Reply:
It gets [set as the default](https://github.com/alphagov/govuk-puppet/blob/260f0efb5b0ab82b4e0b38fd99f20b84972a298b/modules/govuk/manifests/app/envvar/redis.pp#L25) in the upstream, so I didn't think there was required.